app/models/concerns/disbursement_details.rb

```diff
@@ -0,0 +1,25 @@
+module DisbursementDetails
+  extend ActiveSupport::Concern
+
+  included do
+    include DisbursementCosts
+  end
+
+  def translated_disbursement_type
+    if disbursement_type == DisbursementTypes::OTHER.to_s
+      # rubocop:disable Performance/InefficientHashSearch (False positive)
```

Comment:
Why is this a false positive?

Reply:
Rubocop sees `.values` and wrongly concluses that OtherDisbursementTypes is a hash. If it was then `values.include?` would be less efficient than an alternative method of interrogating the values of a hash.

I've made the comment more explicit.

Reply:
Ah and because it's a static analysis tool, it can't know if it's a hash. Makes sense.